engine/source/editor.cpp

```diff
@@ -28,7 +28,9 @@
 #include <imgui/misc/cpp/imgui_stdlib.h>
 
 // TODO: Editor shouldnt depend on this.
```

Comment:
can remove todo. I added it before, but it isnt really something we should concern us with anymore